Sources/DataStructuresAndAlgorithms/Algorithms/SortingAlgorithm.swift

```diff
@@ -131,25 +131,27 @@ struct SortingAlgorithm {
     }
 
     private func merge<T: Comparable>(n1: [T], n2: [T]) -> [T] {
-        var n1 = n1
-        var n2 = n2
+        let size1 = n1.count
```

Comment:
Rename it to `s1` for consistency reasons

Reply:
Done!